wnfs/Cargo.toml

```diff
@@ -28,31 +28,40 @@ bytes = "1.4.0"
 chacha20poly1305 = "0.10"
 chrono = { version = "0.4", default-features = false, features = ["clock", "std"] }
 futures = "0.3"
+insta = { version = "1.30", features = ["json"] }
 libipld-core = { version = "0.16" }
 multihash = "0.19"
 once_cell = "1.16"
 proptest = { version = "1.1", optional = true }
 rand_core = "0.6"
 semver = { version = "1.0", features = ["serde"] }
 serde = { version = "1.0", features = ["rc"] }
-serde_ipld_dagcbor = "0.4.0"
-skip_ratchet = { version = "0.3.0", features = ["serde"] }
+serde-byte-array = "0.1"
+serde_bytes = "0.11"
+serde_ipld_dagcbor = "0.4"
+sha3 = "0.10"
+skip_ratchet = { version = "0.3", features = ["serde"] }
 thiserror = "1.0"
-wnfs-common = { path = "../wnfs-common", version = "0.1.23" }
-wnfs-hamt = { path = "../wnfs-hamt", version = "0.1.23" }
-wnfs-nameaccumulator = { path = "../wnfs-nameaccumulator", version = "0.1.23" }
+wnfs-common = { path = "../wnfs-common", version = "0.1" }
+wnfs-hamt = { path = "../wnfs-hamt", version = "0.1" }
+wnfs-nameaccumulator = { path = "../wnfs-nameaccumulator", version = "0.1" }
```

Comment:
Hm... I'm not sure about this.
What's the reasoning for going for less precise version bounds here? We're not really following semver, so those bounds aren't actually accurate. In fact, we should probably change the version bounds to `version = "= 0.1.23"` because it literally fails to compile otherwise.

Reply:
It fails to compile? I remember zeeshan suggesting keeping patch flexible for all things libraries (can't find the quote right now). That is why our deps have just major.minor versions.

Reply:
Yep I know. But we make breaking changes between e.g. 0.1.21 and 0.1.23 versions.
We recently had [someone in the discord](https://discord.com/channels/478735028319158273/877623827331428403/1136628690147618899) who hit an issue with wnfs 0.1.21 depending on wnfs-common 0.1.23.

Reply:
Eventually we should just get out of version 0.1 & release something that's 1.0 and start doing semver checks.